package.bzl

```diff
@@ -81,11 +80,12 @@ def rules_typescript_dependencies():
         sha256 = "9176a7df34dbed2cf5171eb56271868824560364e60644348219f852f593ae79",
     )
 
-    ###############################################
-    # Repeat the dependencies of rules_nodejs here!
-    # We can't load() from rules_nodejs yet, because we've only just fetched it.
-    # But we also don't want to make users load and call the rules_nodejs_dependencies
-    # function because we can do that for them, mostly hiding the transitive dependency.
+    # io_bazel_rules_webtesting depends on bazel_skylib. It is installed by
+    # web_test_repositories() for for downstream users that are not using
```

Comment:
repeated "for". This sentence is also a little confusing, I think you meant `bazel_skylib` needs to be installed in case users don't call `web_test_repositories()`?

Reply:
Thanks. Cleaned it up.